diff --git a/node_modules/@svar-ui/react-grid/dist/index.es.js b/node_modules/@svar-ui/react-grid/dist/index.es.js
index 81a0428..f48b544 100644
--- a/node_modules/@svar-ui/react-grid/dist/index.es.js
+++ b/node_modules/@svar-ui/react-grid/dist/index.es.js
@@ -1605,9 +1605,9 @@ const Bn = (n) => n.split("-").map((e) => e ? e.charAt(0).toUpperCase() + e.slic
     [D, Le]
   ), Se = Z(Je.theme), ne = I(0);
   return K(() => {
-    if (!ne.current)
+    if (ne.current <= 1) {
       T && T($);
-    else {
+    } else {
       const _ = V.getState();
       V.init({
         data: e,
@@ -1641,7 +1641,7 @@ const Bn = (n) => n.split("-").map((e) => e ? e.charAt(0).toUpperCase() + e.slic
     b,
     T,
     $
-  ]), ne.current === 0 && V.init({
+  ]), ne.current === 0 && (V.init({
     data: e,
     columns: He,
     split: g || { left: 0 },
@@ -1654,7 +1654,7 @@ const Bn = (n) => n.split("-").map((e) => e ? e.charAt(0).toUpperCase() + e.slic
     reorder: E,
     _skin: Se,
     select: b
-  }), Bt(
+  }), ne.current = 1), Bt(
     J,
     () => ({
       ...$
