diff --git a/node_modules/@svar-ui/react-gantt/dist/index.es.js b/node_modules/@svar-ui/react-gantt/dist/index.es.js
index 428e6aa..40eccf1 100644
--- a/node_modules/@svar-ui/react-gantt/dist/index.es.js
+++ b/node_modules/@svar-ui/react-gantt/dist/index.es.js
@@ -1630,36 +1630,41 @@ const fn = (t) => t.split("-").map((s) => s ? s.charAt(0).toUpperCase() + s.slic
   );
   const ke = ee(0);
   Z(() => {
-    ke.current ? F.init({
-      tasks: o,
-      links: v.links,
-      start: m,
-      columns: v.columns,
-      end: P,
-      lengthUnit: T,
-      cellWidth: v.cellWidth,
-      cellHeight: E,
-      scaleHeight: _,
-      scales: v.scales,
-      taskTypes: n,
-      zoom: v.zoom,
-      selected: N,
-      activeTask: h,
-      baselines: C,
-      autoScale: B,
-      unscheduledTasks: O,
-      markers: r,
-      durationUnit: c,
-      criticalPath: J,
-      schedule: j,
-      projectStart: Q,
-      projectEnd: X,
-      calendar: z,
-      undo: U,
-      _weekStart: le.weekStart,
-      splitTasks: R,
-      summary: i
-    }) : a && a(me), ke.current++;
+    if (ke.current <= 1) {
+      a && a(me);
+    } else {
+      F.init({
+        tasks: o,
+        links: v.links,
+        start: m,
+        columns: v.columns,
+        end: P,
+        lengthUnit: T,
+        cellWidth: v.cellWidth,
+        cellHeight: E,
+        scaleHeight: _,
+        scales: v.scales,
+        taskTypes: n,
+        zoom: v.zoom,
+        selected: N,
+        activeTask: h,
+        baselines: C,
+        autoScale: B,
+        unscheduledTasks: O,
+        markers: r,
+        durationUnit: c,
+        criticalPath: J,
+        schedule: j,
+        projectStart: Q,
+        projectEnd: X,
+        calendar: z,
+        undo: U,
+        _weekStart: le.weekStart,
+        splitTasks: R,
+        summary: i
+      });
+    }
+    ke.current++;
   }, [
     me,
     a,
@@ -1688,7 +1693,7 @@ const fn = (t) => t.split("-").map((s) => s ? s.charAt(0).toUpperCase() + s.slic
     R,
     i,
     F
-  ]), ke.current === 0 && F.init({
+  ]), ke.current === 0 && (F.init({
     tasks: o,
     links: v.links,
     start: m,
@@ -1717,7 +1722,7 @@ const fn = (t) => t.split("-").map((s) => s ? s.charAt(0).toUpperCase() + s.slic
     _weekStart: le.weekStart,
     splitTasks: R,
     summary: i
-  });
+  }), ke.current = 1);
   const Te = b(() => z ? (ie, xe) => xe == "day" && !z.getDayHours(ie) || xe == "hour" && !z.getDayHours(ie) ? "wx-weekend" : "" : H, [z, H]);
   return /* @__PURE__ */ d(Se.i18n.Provider, { value: fe, children: /* @__PURE__ */ d(De.Provider, { value: me, children: /* @__PURE__ */ d(
     on,
